.github/workflows/builder.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Build packages'
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 22 * * 2'
+
+jobs:
+  build:
+    name: centos-6
+    runs-on: ubuntu-20.04
```

Comment:
Just to be sure I understand, this will use pdns-builder to run a CentOS 6 docker container from inside an Ubuntu 20.04, right?

Reply:
Yes! and `make check` also runs inside the CentOS 6 container.

Reply:
I'll add some more comments before merging.